vendor/hal_bluetooth_default.te

```diff
@@ -9,7 +9,6 @@ allow hal_bluetooth_default kernel:system module_request;
 allow hal_bluetooth_default self:qipcrtr_socket create_socket_perms_no_ioctl;
 allow hal_bluetooth_default self:socket create;
 
-set_prop(hal_bluetooth_default, wc_prop)
 set_prop(hal_bluetooth_default, bluetooth_prop)
```

Comment:
No complaints about a `set` for a seemingly non-vendor prop here? At least the server is **not**-`neverallow`ed to set it: https://cs.android.com/android/platform/superproject/+/android14-qpr3-release:system/sepolicy/private/property.te;l=257;drc=9f41fc081fd282948ba2257bf025803f5868b8c0

Reply:
good catch. nothing in the log, but I will remove it during rebase